doc/source/performance/threadpool.rst

```diff
@@ -4,19 +4,17 @@
  Thread Pool
 =============
 
+Servers continually execute queries from multiple clients. In *MySQL*, for each connection query, the server creates a thread, processes the query, and then destroys the thread. This method can have disadvantages because the server must consume resources along with the consuming the resources needed to process the query. Therefore, when the number of connections grows the server performance drops. Too many active threads impact performance because of context switching, and thread contention.
 
-This feature enables the server to keep the top performance even with a large
-number of client connections by introducing a dynamic thread pool. By using the
-thread pool server would decrease the number of threads, which will then reduce
-the context switching and hot locks contentions. Using the thread pool will have
-the most effect with ``OLTP`` workloads (relatively short CPU-bound queries).
-
-The |thread-pool| feature introduces a dynamic |thread-pool| that enables the
-server to keep the top performance even with a large number of client
-connections. With |thread-pool| enabled, the server decreases the number of
-threads, which then reduces the context switching and hot locks
-contentions. Using the |thread-pool| has the most effect on ``OLTP`` workloads
-(relatively short CPU-bound queries).
+A thread pool is distinct from connection pooling. A thread pool has the following advantages:
+
+* Limits the number of threads running on the server
+
+* Minimizes wasting resources by creating and then destroying threads
+
+This feature ensures that multiple connections using a thread pool will not cause the server to churn through resources or cause a server exit when the server runs out of memory. A thread pool reuses threads and is most efficient for the short queries associated with transactions. 
+
+If, for example, the server has 100 clients connecting and attempting to run queries, only a fraction of those queries are completed. If the server has a thread pool size of 10 and 100 clients connect to run queries, the thread pool efficiently queues and executes 10 queries at a time in parallel. 
```

Comment:
a fraction of those queries IS completed

Reply:
Redone.